templates/breadcrumb_ec_corporate.component.inc

```diff
@@ -65,3 +65,55 @@ function header_footer_corporate_easy_breadcrumb_breadcrumb_alter(array &$breadc
   }
   $breadcrumb[] = $last_breadcrumb;
 }
+
+/**
+ * Implements hook_process_easy_breadcrumb().
+ */
+function header_footer_corporate_process_easy_breadcrumb(array &$variables) {
+  global $language;
```

Comment:
@serotonine Could you put this variable inside the below condition? There is no need to keep it outside as it's not used.

Reply:
ok